tests/src/StringWithDatetimeFormatValidatorTest.php

```diff
@@ -31,6 +31,7 @@ public function validateSuccessProvider(): array
         //input, expected
         return [
             ['2019-11-14T14:30:26+02:00', '2019-11-14T14:30:26+02:00'],
+            ['2019-11-14T14:30:26+0200', '2019-11-14T14:30:26+0200'],
```

Comment:
`2019-11-14T14:30:26+02`

is also valid

https://tools.ietf.org/html/rfc3339#appendix-A

Reply:
Added. The test case you mention above and `2019-11-14T14:30:26+02`